src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -588,6 +588,10 @@ void prepare_quantization::prepare_asymmetric_quantization(program &p, convoluti
         in1.get_output_layout().data_type != data_types::i8)
         return;
 
+    if (asymmetric_data && in0.is_dynamic() &&
+        in0.get_output_layout().get_partial_shape()[1].get_interval().size() != 1)
```

Comment:
Is it OK to return here? What will happen in such case?

Reply:
Applied assert code.